app/collections/show-list.cjsx

```diff
@@ -140,14 +184,35 @@ module.exports = React.createClass
           meta = @state.subjects[0].getMeta()
 
           <div>
-            <div className="collection-subjects-list">
+          {if @state.selecting
```

Comment:
You can only add or remove from collections if you are signed in, so I don't think these buttons should be visible at all if not signed in.